app/modules/handler/actions/uri.js

```diff
@@ -323,31 +337,27 @@ function unpackTransaction(bytes) {
   return {};
 }
 
-export function signIdentityRequest(
-  prompt,
-  blockchain,
-  wallet,
-) {
+export function signIdentityRequest(prompt, blockchain, wallet) {
   return (dispatch: () => void, getState) => {
-    const {
-      auths,
-      connection,
-      settings,
-      sessions,
-    } = getState();
-    const { enableSessions } = settings;
+    const { auths, connection, settings, sessions } = getState();
+
+    // Always enabling sessions:
+    const enableSessions = true;
```

Comment:
Setting it here.

Reply:
This might as well be removed, since it's only used once in an `if` statement. Remove it here and just remove it as a part of the `if` statement.